CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## MASTER
 
 ## Dotenv 0.9.0.1
+### Added
```

Comment:
Should we add in here that the support for some 8.x versions was dropped and we added 9.0.1?

Reply:
Mmm that's a good one. Thanks!